src/helpers/graph.js

```diff
@@ -1,38 +1,45 @@
-export function removeNoteFromGraph(nodes, edges, noteToDelete) {
+function doesDeletedNodeHasSiblings(edges, parentId) {

```

Comment:
Why *deleted* node? Doesn't it work for any node?

Also, why can't we do it like `doesNodeHasSiblings(edges, nodeId)` ? Why do we have to pass the parentId? I think it's easy to find a parent knowing the nodeId?

Reply:
Done